projects/dsp-ui/src/lib/search/expert-search/expert-search.component.ts

```diff
@@ -35,9 +35,11 @@ export class ExpertSearchComponent implements OnInit {
     expertSearchForm: FormGroup;
     queryFormControl: FormControl;
 
+    iriBaseUrl = this.getIriBaseUrl();
```

Comment:
I think this property is redundant, since you do not use it on view, then it can be removed and in line 42 `getIriBaseUrl()` method could be directly used.

Reply:
Yes maybe. But I thought to move it to js-lib one day similar to the already defined `get apiUrl`. I will create a corresponding user story...

Reply:
Done in DSP-1036